### ISSUE TITLE
Is there a reason for not supporting parseDates in useCollectionGroup?

### ISSUE THREAD

**Issue by manuganji** [nandorojo/swr-firestore#83]
Hi, I use parseDates in useDocument and useCollection. How can I do it with useCollectionGroup? Is there a reason for not supporting it?

**Comment by nandorojo** [comment]
I think it should work the same, does it not?

**Comment by manuganji** [comment]
Hey @nandorojo, thanks for the quick reply. I believe it's not showing in the type signature. Let me try again with a `//@ts-ignore`

**Comment by nandorojo** [comment]
You're right, the types are missing.

You can also use `useCollection` and pass `isCollectionGroup: true`.

**Comment by manuganji** [comment]
I went with your second solution. Thank you very much @nandorojo! It worked! :)
